hw1.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Cubes</title>
+  </head>
+  <body>
+    <div class="row" id=widget>
+        <div class="col" id="Chicago"><img src="chicago.png" class="rounded mx-auto d-block" alt="rain"> </br> Chicago / 23°</div>
+        <div class="col" id="Seattle"><img src="Siettle.png" class="rounded mx-auto d-block" alt="fog"> </br> Seattle / 34°</div>
+        <div class="col" id="Brooklyn"><img src="brooklyn.png" class="rounded mx-auto d-block" alt="clouds and sun"></br> Brooklyn / 55°</div>
```

Comment:
Using `<br>` is not considered best practice, with the component structure described above, styles can be added to the `<div>` wrapper (margins, paddings) to eliminate the `<br>`.